ARTnews.js

```diff
@@ -0,0 +1,155 @@
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2017 czar
+	http://en.wikipedia.org/wiki/User_talk:Czar
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+ 
+function detectWeb(doc, url) { return FW.detectWeb(doc, url); }
+function doWeb(doc, url) { return FW.doWeb(doc, url); }
+ 
+FW.Scraper({
+itemType         : 'blogPost',
+detect           : FW.Url().match(/\/\d{4}\/\d{2}\//),
+title            : FW.Xpath('//h1[@class="entry-title"]').text(),
+attachments      : [{ url: FW.Url(),
+  title:  "ARTnews snapshot",
+  type: "text/html" }],
+creators         : FW.Xpath('//span[@class="author-meta-name"]/a').text().cleanAuthor("author"),
+date             : FW.Xpath('//span[@class="date-meta"]').text().replace(/(\d{2})\/(\d{2})\/(\d{2})/, "20$3-$1-$2,"), // "03/02/16 10:35 am" => 2017-03-02, 10:35 pm
+blogTitle        : "ARTnews",
+language         : "en-US",
+abstractNote     : FW.Xpath('//meta[@name="description"]/@content').text(),
+tags             : FW.Xpath('//span[@class="cat-links"]/a').text(),
+hooks : { "scraperDone": function  (item,doc,url) {
+	if (item.creators[0].lastName == "ARTnews") {
+		item.creators[0].lastName = "The Editors of ARTnews";
```

Comment:
you need to remove the `firstName` here still (I think you can just do that with `delete` but I always get confused about removing items from objects.